src/convert/markdown-plugins/index.ts

```
@@ -0,0 +1,214 @@
+
+  register(plugin: MarkdownPlugin): void {
+    this.plugins.set(plugin.name, plugin);
+  }
+
+  get(name: string): MarkdownPlugin | undefined {
+    return this.plugins.get(name) || BUILTIN_PLUGINS[name];
+  }
+
+  getAll(): MarkdownPlugin[] {
+    const all = new Map(Object.entries(BUILTIN_PLUGINS));
+    this.plugins.forEach((plugin, name) => all.set(name, plugin));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)